WireAPI/Tests/WireAPITests/Helpers/XCTestCase+Assertions.swift

```diff
@@ -20,41 +20,70 @@ import XCTest
 
 extension XCTestCase {
 
-    /// Assert that a error is thrown when a block is performed.
+    /// Assert that an error is thrown when a block is performed.
```

Comment:
is this not a bit confusing compared to the one provided by Apple?
```
XCTAssertThrowsError<T>(_ expression: @autoclosure () throws -> T, _ message: @autoclosure () -> String = "", file: StaticString = #filePath, line: UInt = #line, _ errorHandler: (_ error: any Error) -> Void = { _ in })
```

Reply:
After discussion with @markusfassbender we thought it would be better to stay closer to Apple API because it's familiar. Are you concerned that the compiler won't pick the right version? This is an `async` variant of the XCTest assertion so I think it should be fine.

Reply:
alright thks for the clarification